odl/tomo/__init__.py

```diff
@@ -26,3 +25,11 @@
 from . import backends
 from .backends import *
 __all__ += backends.__all__
+
+from . import geometry
```

Comment:
There should be ordered properly. particularily, I guess backends depend on geometry and not the other way around?